### DISCUSSION TITLE
Severity values not consistent

### DISCUSSION THREAD

**Discussion by geiregjo** [primefaces/community#2642] [PrimeVue]
I'm using PrimeVue as a base for a component library. I created a type Severity which I intended to use everywhere. Then I noticed that it's values are not consistent in PrimeVue. For instance, [Button](https://primevue.org/button/#api.button.props.severity) has severity "danger" but [Toast](https://primevue.org/toast/#api.toast.interfaces.ToastMessageOptions.severity) it's equivalent is "error"? Is this an oversight? If not, what's the intention?

**Comment by leosin** [comment]
This problem also bothers me, and the `Message` component also use `error`

**Comment by Maxime-BERGEON-SADA** [comment]
Bumping this issue as this is clearly misleading
